src/app/bug/shared/bug.service.spec.ts

```diff
@@ -15,19 +21,50 @@ describe('BugService', () => {
     ]
   });
 
+  const createFakeBugModel = (): Bug => {
+    let bug = {
+      name: faker.random.word(),
+      location: faker.random.word(),
+      time: faker.random.word(),
+      price: faker.datatype.number(5),
+      month:{
+        north: [],
+        south: []
+      } as Month,
+      caught:false
+    } as Bug;
+    return bug;
+  }
+
+  const createFakeBugArray = (): Bug[] => {
+    let counter = 5;
+    let bugArray: Bug[] = [];
+    for(let i =0;i<counter;i++){
+        bugArray.push(createFakeBugModel());
+    }
+    return bugArray;
+  }
+  
+  
   beforeEach(() => {
+    data = createFakeBugArray();
     spectator = createService();
     service = spectator.service;
+    
+    mockDataService = spectator.inject(BugDataService);
   });
 
   it('should be created', () => {
-    expect(service).toBeTruthy();
+    expect(spectator.service).toBeTruthy();
   });
 
-  /** 
   describe('getBugsData', () => {
-    it('should update the state', () => {
-      expect(spectator.component.state).toBeEmpty;
-    })
-  })*/
+    
+    it('should get the bug data from the data service', () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/JohnnyGitH/BugEDex/pullRequest?prid=8845963)
```suggestion
    it("should get the bug data from the data service", () => {
```